### DISCUSSION TITLE
not able to change brightness.

### DISCUSSION THREAD

**Discussion by Khishge** [seerge/g-helper#1474] [Q&A] [answered]
zephyrusg14 2022 RYZEN9 6900 / RX 6700
when the gpu mode is on eco and standard i am not able to adjust my brightness. Ive read on reddit that it might be something to do with power modes so i checked it and it was on "best power efficienty" changed it didnt solve anything(and i don't even know whether it is changing or not since nothing happening). I could go into the ultimate gpu mode to adjust brightness but it is not convinient since it require restart.
few things i noticed : 
    - once im in the ultimate gpu mode after restarting the power mode is automatically on "Best performance"
    - in the eco and the standard gpu mode the most  refresh rate i can change to (in the G-helper Laptop screen section) is "64Hz + OD" while in the ultimate gpu mode i can get "120Hz + OD"
 
If i remember correctly yesterday i was able to adjust my brightness and set refresh rate to "120Hz + OD" in any mode.

any help would be much appreciated !

**Comment by seerge** [comment, marked as answer]
@Khishge hello, it looks like your GPU is not turning or or off between eco/standard :) Set standard mode and check device manager if everything is ok there. 

``64Hz`` is usually a sign of absent GPU driver, so may be time to do a fresh resinstall of latest amd driver using https://www.amd.com/en/support/kb/faq/gpu-601 ?

**Reply by Khishge** [reply]
Drive reinstallation worked ! 

Thank you for your prompt response and for taking the time to help me. I really appreciate it.
